### PR TITLE
Buffs salvaged PA move speed slightly, nerfs NCR PA, removes raider/hotrod PA.

### PR DESCRIPTION
**Pull request by Mannybrado**
## About The Pull Request
title

## Why It's Good For The Game
makes salvaged PA more specialized without being stupidly inconsistent between versions. also, raider/hotrod PA are ugly.

## Changelog
:cl:
changelogs dont work
/:cl:

### CODE REVIEW

**Comment by panzer1944** [comment]
ok but why remove hotrod and raider.
I can understand them becoming craft-only, but straight up removing them fully is hghbhhghhh, especially when the reasoning is just `ugly` - just keep their values somewhat consistent, maybe make hotrod a bit faster at the cost of protection, and have raider just be a reskin, idk

**Comment by Wirpy** [comment]
Don't remove hotrod/raider PA, they're pretty cool.

File: code/_globalvars/lists/maintenance_loot.dm [hunk 1]
```diff
@@ -393,10 +393,6 @@ GLOBAL_LIST_INIT(loot_t4_armor, list(
 	/obj/item/clothing/head/helmet/f13/rangerbroken,
 	/obj/item/clothing/suit/armor/f13/combat/mk2,
 	/obj/item/clothing/head/helmet/f13/combat/mk2,
```
Comment (Looreydev) [inline code/_globalvars/lists/maintenance_loot.dm:395]:
```suggestion
	/obj/item/clothing/head/helmet/f13/combat/mk2
```
Remove the comma

File: code/game/objects/effects/spawners/f13lootdrop.dm [hunk 2]
```diff
@@ -374,8 +374,6 @@
 				/obj/effect/spawner/bundle/f13/armor/riot,
 				/obj/effect/spawner/bundle/f13/armor/combat/mk2,
 				/obj/effect/spawner/bundle/f13/armor/combat/mk2/dark,
```
Comment (Looreydev) [inline code/game/objects/effects/spawners/f13lootdrop.dm:376]:
```suggestion
				/obj/effect/spawner/bundle/f13/armor/combat/mk2/dark
```
Same here, remove comma

File: code/modules/fallout/misc/gangs.dm [hunk 3]
```diff
@@ -97,9 +97,6 @@ GLOBAL_DATUM_INIT(denmob, /datum/gang/denmob, new)
 		/datum/gang_item/clothing/glasses/sunglasses,
 		/datum/gang_item/weapon/type17,
 		/datum/gang_item/weapon/type93,
```
Comment (Looreydev) [inline code/modules/fallout/misc/gangs.dm:99]:
```suggestion
		/datum/gang_item/weapon/type93
```
Remove this comma too

**Review by Looreydev — CHANGES_REQUESTED** [review]

File: code/game/objects/effects/spawners/f13lootdrop.dm (hunk 2 above)
Comment (Mannybrado) [reply code/game/objects/effects/spawners/f13lootdrop.dm]: done

File: code/_globalvars/lists/maintenance_loot.dm (hunk 1 above)
Comment (Mannybrado) [reply code/_globalvars/lists/maintenance_loot.dm]: done

File: code/modules/fallout/misc/gangs.dm (hunk 3 above)
Comment (Mannybrado) [reply code/modules/fallout/misc/gangs.dm]: done

**Review by Looreydev — APPROVED** [review]

**Comment by Mannybrado** [comment]
man. fuck it.

**Pull request closed without merging**